docs/core/tutorials/using-on-windows.md

```diff
@@ -107,7 +107,7 @@ A solution using only .NET Core projects
         Assert.Equal(42, new Thing().Get(42));
     }
     ```
-    * Optionally, delete the `Program.cs` file from the **TestLibrary** project, and remove `"buildOptions": {"emitEntryPoint": true}` from `project.json`.
+    * Optionally, delete the `Program.cs` file from the **TestLibrary** project, and remove `"buildOptions": {"emitEntryPoint": true,` from `project.json`.
```

Comment:
@svick  Thanks for catching this. A byproduct of cut and paste not working quite right on the new site.